crates/rust-analyzer/src/config.rs

```diff
@@ -494,6 +494,9 @@ config_data! {
         /// Exclude imports from find-all-references.
         references_excludeImports: bool = "false",
 
+        /// Allow renaming of items not belonging to any workspace crates.
+        renameExternalItems_enable : bool = "false",
```

Comment:
Might be better to add the `allow` part to the name, and then to future proof namespace it so `rename_allowExternalItems` or something allong those lines?